crates/jstz_cli/src/logs/mod.rs

```diff
@@ -9,36 +11,29 @@ mod trace;
 pub enum Command {
     /// View logs
     Trace {
-        /// View console.log messages
-        #[arg(long)]
-        log: bool,
-        /// View console.info messages
-        #[arg(long)]
-        info: bool,
-        /// View console.warn messages
-        #[arg(long)]
-        warn: bool,
-        /// View console.error messages
-        #[arg(long)]
-        error: bool,
-        /// View contract creations
-        #[arg(long)]
-        contract: bool,
-        /// Add custom search strings
-        #[arg(long, value_name = "custom_strings")]
-        custom: Vec<String>,
+        // The address or the alias of the smart function
+        #[arg(value_name = "ALIAS|ADDRESS")]
+        smart_function: String,
     },
 }
 
-pub fn exec(command: Command, cfg: &mut Config) -> Result<()> {
+pub async fn exec(command: Command, cfg: &mut Config) -> Result<()> {
     match command {
-        Command::Trace {
-            log,
-            info,
-            warn,
-            error,
-            contract,
-            custom,
-        } => trace::exec(log, info, warn, error, contract, custom, cfg),
+        Command::Trace { smart_function } => {
+            let address = get_address_for_smart_function(&smart_function, cfg)?;
+            trace::exec(address, cfg).await
+        }
     }
 }
+
+fn get_address_for_smart_function(smart_function: &str, cfg: &Config) -> Result<Address> {
+    if let Ok(address) = PublicKeyHash::from_base58(smart_function) {
+        return Ok(address);
+    }
+
+    if let Ok(account) = cfg.accounts.get(smart_function) {
+        return Ok(account.address().clone());
+    }
+
+    Err(anyhow!("Invalid smart function alias or address"))
+}
```

Comment:
@alanmarkoTrilitech I think we have logic for this elsewhere in the CLI. Could @ryutamago use it?

Reply:
I don't think there is exactly this variant where you get address from either account or alias - there are methods to get either an account from alias or use the self-address one. I think @ryutamago can move this method to AccountConfig in jstz_cli/config.rs and make it general so that it does not apply just to smart function addresses (it can be named e.g. just get_address)